frontend/src/DropdownMenu/DropdownMenu.tsx

```diff
@@ -40,7 +42,10 @@ function DropdownMenu(props: IDropdownMenuProps) {
   const theme = useTheme();
   const buttonText = {
     signin: (
-      <strong style={{ color: theme.palette.primary.main }}>SIGN IN </strong>
+      <SigninForm signInClick={signInClick}>
```

Comment:
wrapping the sign-in button in the `SigninForm` with a dialog is an interesting solution - pretty unorthodox but it works lol

Reply:
usually the sign-in button would trigger a callback prop, then the parent component (App.tsx) would render the sign-in form as its child (and sibling of DropdownMenu.tsx)

but 👍 for creative solution